ocs_ci/ocs/ui/add_replace_device_ui.py

```diff
@@ -30,9 +30,12 @@ def add_capacity_ui(self):
             self.do_click(self.add_capacity_ui_loc["ocs_operator"])
             self.do_click(self.add_capacity_ui_loc["storage_cluster_tab"])
         time.sleep(1)
+        logger.info("Click on kebab menu of Storage Systems")
         self.do_click(
             self.add_capacity_ui_loc["kebab_storage_cluster"], avoid_stale=True
         )
+        self.take_screenshot()
```

Comment:
we may provide description as an arg to self.take_screenshot. This way it is easier to find correct screenshot, not guessing and calculating the capture time.

Reply:
sure, it can be done later if necessary